### ISSUE TITLE
Levers fall off of walls when reloading a DecayDungeon

### ISSUE THREAD

**Issue by Ameliorate** [CivWizardry/DungeonMod#16]

**Comment by Ameliorate** [comment]
Oh, here's the smoking gun of the cause:
```
[16:46:23 ERROR]: Error occurred while enabling DungeonMod v2.1.0-@apiversion@ (Is it up to date?)
java.lang.IllegalArgumentException: Provided data is not of type org.bukkit.material.MaterialData, found org.bukkit.material.Lever
	at org.bukkit.craftbukkit.v1_12_R1.block.CraftBlockState.setData(CraftBlockState.java:94) ~[spigot-1.12.2.jar:git-Spigot-e8ded36-acbc348]
	at pw.amel.dungeonmod.blockcopy.MetaCopier.copy(MetaCopier.java:9) ~[?:?]
	at pw.amel.dungeonmod.blockcopy.CopyBlock.copyBlock(CopyBlock.java:37) ~[?:?]
	at pw.amel.dungeonmod.DecayDungeon.rebuild(DecayDungeon.java:82) ~[?:?]
	at pw.amel.dungeonmod.DecayDungeon.<init>(DecayDungeon.java:51) ~[?:?]
	at pw.amel.dungeonmod.ConfigManager.lambda$doDungeonsSection$0(ConfigManager.java:65) ~[?:?]
	at java.util.LinkedHashMap.forEach(LinkedHashMap.java:684) ~[?:1.8.0_181]
	at pw.amel.dungeonmod.ConfigManager.doDungeonsSection(ConfigManager.java:32) ~[?:?]
	at pw.amel.dungeonmod.ConfigManager.reload(ConfigManager.java:26) ~[?:?]
	at pw.amel.dungeonmod.DungeonMod.onEnable(DungeonMod.java:34) ~[?:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:264) ~[spigot-1.12.2.jar:git-Spigot-e8ded36-acbc348]
	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:337) [spigot-1.12.2.jar:git-Spigot-e8ded36-acbc348]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:403) [spigot-1.12.2.jar:git-Spigot-e8ded36-acbc348]
	at org.bukkit.craftbukkit.v1_12_R1.CraftServer.enablePlugin(CraftServer.java:381) [spigot-1.12.2.jar:git-Spigot-e8ded36-acbc348]
	at org.bukkit.craftbukkit.v1_12_R1.CraftServer.enablePlugins(CraftServer.java:330) [spigot-1.12.2.jar:git-Spigot-e8ded36-acbc348]
	at org.bukkit.craftbukkit.v1_12_R1.CraftServer.reload(CraftServer.java:752) [spigot-1.12.2.jar:git-Spigot-e8ded36-acbc348]
	at org.bukkit.Bukkit.reload(Bukkit.java:525) [spigot-1.12.2.jar:git-Spigot-e8ded36-acbc348]
	at org.bukkit.command.defaults.ReloadCommand.execute(ReloadCommand.java:27) [spigot-1.12.2.jar:git-Spigot-e8ded36-acbc348]
	at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:141) [spigot-1.12.2.jar:git-Spigot-e8ded36-acbc348]
	at org.bukkit.craftbukkit.v1_12_R1.CraftServer.dispatchCommand(CraftServer.java:648) [spigot-1.12.2.jar:git-Spigot-e8ded36-acbc348]
	at net.minecraft.server.v1_12_R1.PlayerConnection.handleCommand(PlayerConnection.java:1401) [spigot-1.12.2.jar:git-Spigot-e8ded36-acbc348]
	at net.minecraft.server.v1_12_R1.PlayerConnection.a(PlayerConnection.java:1236) [spigot-1.12.2.jar:git-Spigot-e8ded36-acbc348]
	at net.minecraft.server.v1_12_R1.PacketPlayInChat.a(PacketPlayInChat.java:45) [spigot-1.12.2.jar:git-Spigot-e8ded36-acbc348]
	at net.minecraft.server.v1_12_R1.PacketPlayInChat.a(PacketPlayInChat.java:1) [spigot-1.12.2.jar:git-Spigot-e8ded36-acbc348]
	at net.minecraft.server.v1_12_R1.PlayerConnectionUtils$1.run(SourceFile:13) [spigot-1.12.2.jar:git-Spigot-e8ded36-acbc348]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_181]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_181]
	at net.minecraft.server.v1_12_R1.SystemUtils.a(SourceFile:46) [spigot-1.12.2.jar:git-Spigot-e8ded36-acbc348]
	at net.minecraft.server.v1_12_R1.MinecraftServer.D(MinecraftServer.java:748) [spigot-1.12.2.jar:git-Spigot-e8ded36-acbc348]
	at net.minecraft.server.v1_12_R1.DedicatedServer.D(DedicatedServer.java:406) [spigot-1.12.2.jar:git-Spigot-e8ded36-acbc348]
	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:679) [spigot-1.12.2.jar:git-Spigot-e8ded36-acbc348]
	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:577) [spigot-1.12.2.jar:git-Spigot-e8ded36-acbc348]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_181]
```

**Comment by Ameliorate** [comment]
This is caused by levers not being handled in BlockCopy. #9 would solve this problem much more elegantly compared to just fixing this issue in isolation.

**Comment by Ameliorate** [comment]
Fixed by 15503bba4cab8ed08913a75a722cdbebd2ffab15
